jmclient/jmclient/wallet_rpc.py

```diff
@@ -420,7 +423,8 @@ def dummy_restart_callback(msg):
                         walletname=self.wallet_name,
                         token=self.cookie)
 
-    def taker_finished(self, res, fromtx=False, waittime=0.0, txdetails=None):
+    def taker_finished(self, res, fromtx=False,
+                               waittime=0.0, txdetails=None):
```

Comment:
On these indentation comments: I don't want to bother changing my workflow right now to do what is needed to make it a specific thing (whatever specifically that your workflow/environment requires). I'll just accept your suggested changes.

Reply:
Yeah, I know we don't bother about that, I commented those specifically because they are unnecessary changes that worsen the PEP style, if it were old code or changed for specific reasons I'd try not to bother (something may slip through, I check for those things automatically, sorry if it's an annoyance).